doc/en/contents.rst

```diff
@@ -7,38 +7,65 @@ Full pytest documentation
 
 .. `Download latest version as EPUB <http://media.readthedocs.org/epub/pytest/latest/pytest.epub>`_
 
+
+Start here
+-----------
+
 .. toctree::
    :maxdepth: 2
 
    getting-started
+
+
+How-to guides
+-------------
+
+.. toctree::
+   :maxdepth: 2
+
    usage
    existingtestsuite
    assert
-   fixture
    mark
    monkeypatch
    tmpdir
    capture
-   warnings
-   doctest
    skipping
    parametrize
+   plugins
+   nose
+   bash-completion
+
+
+Reference guides
+-----------------
+
+.. toctree::
+   :maxdepth: 2
+
+   fixture
```

Comment:
Perhaps this should be part of the "How-to" guides, given how fundamental fixtures are?

Reply:
@nicoddemus I have big plans for `fixture.rst`; it's way too large and contains multiple different kinds of things, so I plan to split it up and redistribute its contents in the PR coming after next.

Reply:
Great, sounds good!